google/cloud/bigquery/table.py

```diff
@@ -2215,6 +2279,69 @@ def to_dataframe(
         if time_dtype is DefaultPandasDTypes.TIME_DTYPE:
             time_dtype = db_dtypes.TimeDtype()
 
+        if range_date_dtype is DefaultPandasDTypes.RANGE_DATE_DTYPE:
+            try:
+                range_date_dtype = pandas.ArrowDtype(
+                    pyarrow.struct(
+                        [("start", pyarrow.date32()), ("end", pyarrow.date32())]
+                    )
+                )
+            except AttributeError:
+                # pandas.ArrowDtype was introduced in pandas 1.5, but python 3.7
+                # only supports upto pandas 1.3. If pandas.ArrowDtype is not
+                # present, we raise a warning and set range_date_dtype to None.
+                msg = (
+                    "Unable ro find class ArrowDtype in pandas, setting "
```

Comment:
s/ro/to/ here and in the other two msgs below

Reply:
Thanks for catching this!